jme3-android/src/main/java/com/jme3/system/android/JmeAndroidSystem.java

```diff
@@ -35,6 +37,18 @@ public class JmeAndroidSystem extends JmeSystemDelegate {
         } catch (UnsatisfiedLinkError e) {
         }
     }
+
+    public JmeAndroidSystem(){
+        setErrorMessageHandler((message) -> {
+            String finalMsg = message;
+            String finalTitle = "Error in application";
+            Context context = JmeAndroidSystem.getView().getContext();
+            view.getHandler().post(() -> {
+                AlertDialog dialog = new AlertDialog.Builder(context).setTitle(finalTitle).setMessage(finalMsg).create();
+                dialog.show();
+            });
+        });
+    }
```

Comment:
Shouldn't there be a settings handler also provided here (that does nothing and only returns true like the old behavior)?

Reply:
I don't think so,the default implementation will do nothing if jme-awt-dialogs is not included, by leaving it  like this there could be a module implementing settings dialogs for ios and android.

Reply:
Ok